src/renderer/components/execute/ParameterContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import { Form, InputGroup } from 'react-bootstrap';
+import { ProcedureParameter } from 'db/Procedures';
+
+interface IParameterContainerProps {
+  activeParameters: ProcedureParameter[];
+  handleInput: (inputValue: string, attribute: string) => void;
+}
+
+const ParameterContainer = ({
+  activeParameters,
+  handleInput,
+}: IParameterContainerProps) => {
+  if (activeParameters.length > 0) {
+    return (
+      <div className="condition-box">
+        <h4> Set Procedure Parameters </h4>
+        {activeParameters.map(
+          (item) =>
+            item.direction === 'IN' && (
+              <InputGroup className="mb-2" key={`${item.name}`}>
+                <InputGroup.Text bsPrefix="paramater-text">
```

Comment:
```suggestion
                <InputGroup.Text>
```
forgot to take this out whoops